src/components/BrightcovePlayer.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react';
+import './BrightcovePlayer.css';
+import {connect} from "react-redux";
+import Player from '@brightcove/react-player-loader';
+
+class BrightcovePlayer extends Component {
+
+    success = ({ ref }) => {
+        this.playerRef = ref;
+        if (this.props.selectedVideo !== null) {
+            this.playerRef.catalog.load({
+                sources: [this.props.selectedVideo]
```

Comment:
`selectedVideo` is a videoId right? Is this the sanctioned way of doing this?

Reply:
Sure, this seems fine.